security-analysis/security-analysis-api/src/test/resources/SecurityAnalysisParameters.json

```diff
@@ -1,5 +1,10 @@
 {
-  "version" : "1.0",
+  "version" : "1.1",
+  "increasedFlowViolationsThreshold" : "0.1",
```

Comment:
We should use plain floats here, not strings

Reply:
Done